server.php

```diff
@@ -0,0 +1,78 @@
+<?php
+session_start();
+$message = "";
+$username = "";
+$email = "";
+$errors = array();
+$db = mysqli_connect('localhost', 'root', 'abiit@2019', 'rconnect');
+if (mysqli_connect_errno()) {
+    echo "Failed to connect to MySQL: " . mysqli_connect_error();
+    exit();
+  }
+ 
+  $all_users = mysqli_query($db, "SELECT * FROM userData;");
+  $user_count = mysqli_num_rows($all_users);
+if(isset($_POST['reg_user'])){
+    $username = mysqli_real_escape_string($db, $_POST['username']);
+    $email = mysqli_real_escape_string($db, $_POST['email']);
+    $password_1 = mysqli_real_escape_string($db, $_POST['password_1']);
+    $password_2 = mysqli_real_escape_string($db, $_POST['password_2']); 
+    $name = mysqli_real_escape_string($db, $_POST['name']);
+    $enrl = mysqli_real_escape_string($db, $_POST['enrl']);
+    $bhawan = mysqli_real_escape_string($db, $_POST['bhawan']);
+
+if (empty($username)) { array_push($errors, "Username is required"); }
+if (empty($email)) { array_push($errors, "Email is required"); }
+if (empty($password_1)) { array_push($errors, "Password is required"); }
+if ($password_1 != $password_2) {array_push($errors, "The two passwords do not match"); }
```

Comment:
Just a side note : slack comparison is not secure and can lead to complications.
Look up PHP type juggling vulnerability

Reply:
Sure!